README.md

```diff
@@ -11,46 +11,65 @@ This template includes:
 - BitBucket pipelines for deploying production and staging
 - Localstack packages for local development
 - Dockerised local development so you don't have to install the serverless client or localstack
+- Step though debugging support for VSCode
 
 # Usage
 
 ## Setup
 
 ### Aliased NPM
 
+#### Prerequisite
+- Docker
+- Docker-compose
+- (Optional) Visual Studio Code with [Docker extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) installed.
+
 This ensures all serverless commands are run inside the serverless docker container so that you don't need to install node, npm and serverless globally.
 
-Add the following to your `.bashrc` file:
+Add the following to your `.bashrc` or `.zshrc` file:
 
-```
+```bash
+# Normal aliases for normal usage & interacting with online AWS
 alias node-run='docker run --rm -it --volume ~/.aws:/home/node/.aws --volume ~/.npm:/home/node/.npm --volume $PWD:/app aligent/serverless'
 alias serverless='node-run serverless'
-alias sls-deploy-local='docker-compose exec -u node -w /app offline /serverless/node_modules/serverless/bin/serverless.js deploy --verbose --aws-profile localstack --stage dev'
-alias sls-invoke='docker-compose exec -u node -w /app offline /serverless/node_modules/serverless/bin/serverless.js invoke --verbose --aws-profile localstack --stage dev --function'
-alias sls-invoke-stepf='docker-compose exec -u node -w /app offline /serverless/node_modules/serverless/bin/serverless.js invoke stepf --verbose --aws-profile localstack --stage dev --name'
+alias npm='node-run npm'
```

Comment:
We shouldn't have this - some devs have local npm installed for FE work.

Reply:
I removed this `npm` alias & fixed the issue `docker: Error response from daemon: manifest for aligent/serverless:debug not found`